src/main/resources/prl-harvester_messages.xml

```diff
@@ -10,5 +10,6 @@
   <entry key="PRL_002">Missing value for field '{}'</entry>
   <entry key="PRL_003">Missing at least one of 'email', 'phone', or 'webContact'</entry>
   <entry key="PRL_004">Invalid value for field '{}': {}</entry>
+  <entry key="PRL_050">User '{}' connecting to test database at port '{}' with password: {}</entry>
```

Comment:
I think you want this one to be PRL_005, not PRL_050.

Reply:
I can change but I selected a number ahead because Mark was working on a PR at the same time and I didn't want to step on his 005 (if he had one). I'll change now though since his is merged and there was no 005. :-)